filecoin-proofs/tests/api.rs

```diff
@@ -1923,7 +1946,9 @@ fn generate_proof<Tree: 'static + MerkleTreeTrait>(
     seed: [u8; 32],
     pre_commit_output: &SealPreCommitOutput,
     piece_infos: &[PieceInfo],
+    aggregation_enabled: bool,
```

Comment:
This is optional, but I had some difficulty following where exactly aggregation was happening in the tests, maybe this parameter could be named something like `return_groth16_proofs` or `return_ni_porep_groth16_proofs`?

`return_groth16_proofs` would always be `true` when running interactive PoRep (because `seal_commit_phase2` always returns Groth16 proofs for interactive PoRep) and when `true` for NI-PoRep `generate_proof` returns Groth16 proofs (conversely when `false`, `generate_proof` returns the aggregate proof generated by `seal_commit_phase2`).

Reply:
I'll leave this as-is, since it doesn't affect the exposed API

Reply:
Sounds good to me.